docs/en/observability/apm/open-telemetry.asciidoc

```diff
@@ -1,56 +1,147 @@
+which OpenTelemetry API features are supported in the APM agent documentation:
+
+* {apm-java-ref}/opentelemetry-bridge.html[*APM Java agent →*]
+* {apm-dotnet-ref}/opentelemetry-bridge.html[*APM .NET agent →*]
+* {apm-node-ref}/opentelemetry-bridge.html[*APM Node.js agent →*]
+* {apm-py-ref}/opentelemetry-bridge.html[*APM Python agent →*]
+
+[float]
+[[apm-otel-upstream]]
+== Upstream OpenTelemetry language SDK or Collector
 
 The {stack} natively supports the OpenTelemetry protocol (OTLP).
 This means trace data and metrics collected from your applications and infrastructure by an
-OpenTelemetry collector can be sent directly to the {stack}.
+OpenTelemetry language SDK or OpenTelemetry Collector can be sent directly to the {stack}.
+
+image::images/apm-otel-api-sdk-collector.png[]
+
+// Why you _would_ choose this approach
+// OpenTelemetry agent
+If you want to instrument a technology that Elastic doesn't provide a solution for,
+you can use an upstream OpenTelemetry language SDK to send data directly to APM Server.
+For example, if you want to instrument C or C++ you could use the
+https://github.com/open-telemetry/opentelemetry-cpp[OpenTelemetry C{plus}{plus} client].
+// Other languages include erlang, lua, perl.
+
+// Why you _would_ choose this approach
+// OpenTelemetry Collector
+Alternatively, you can use an OpenTelemetry Collector to collect data and send it directly to APM Server without maintaining multiple agents.
 
-<<apm-connect-open-telemetry-collector,Get started>>.
+// Why you would _not_ choose this approach
+However, there are some limitations when using language SDKs and collectors built and maintained
+by OpenTelemetry, including:
 
-**Lambda collector exporter**
+* Elastic can't provide implementation support on how to use upstream OpenTelemetry tools.
+* You won't have access to Elastic enterprise APM features.
+* You may experience problems with performance efficiency.
+
+For more on the limitations associated with using upstream OpenTelemetry tools,
+refer to <<apm-open-telemetry-known-limitations>>.
+
+// Where to go next
+<<apm-open-telemetry-direct,*Get started with upstream OpenTelemetry SDKs and Collectors →*>>
+
+[float]
+[[apm-otel-lambda]]
+== AWS Lambda collector exporter
 
 AWS Lambda functions can be instrumented with OpenTelemetry and monitored with Elastic {observability}.
 
-<<apm-open-telemetry-aws-lambda,Get started>>.
+// Do we want to say anything about why you would/wouldn't choose this
+// method to send data to Elastic?
+
+// Where to go next
+To get started, follow the official AWS Distro for OpenTelemetry Lambda documentation, and configure the OpenTelemetry Collector to output traces and metrics to your Elastic cluster:
+
+https://aws-otel.github.io/docs/getting-started/lambda[*Get started with the AWS Distro for OpenTelemetry Lambda*^] image:images/icons/popout.svg[]
+
+// TODO: I'm not sure what this diagram should look like...
+// image::images/apm-otel-lambda.png[]
```

Comment:
I don't feel confident enough about this diagram to include it. I can hide it for now unless there's someone who can [take a closer look](https://github.com/colleenmcginnis/observability-docs/blob/apm-otel-note/docs/en/observability/apm/images/apm-otel-lambda.png).

Reply:
Agreed. I don't have enough knowledge here to help. Let's not let this block though. This is something we can work with @estolfo on perfecting.